tx/textual/valuerenderer/duration.go

```diff
@@ -0,0 +1,217 @@
+package valuerenderer
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+	dpb "google.golang.org/protobuf/types/known/durationpb"
+)
+
+type durationValueRenderer struct{}
+
+// NewDurationValueRenderer returns a ValueRenderer for protocol buffer Duration messages.
+// It renders durations by grouping seconds into units of days (86400s), hours (3600s),
+// and minutes(60s), plus the total seconds elapsed. E.g. a duration of 1483530s is
+// formatted as "17 days, 4 hours, 5 minutes, 30 seconds (1483530 seconds total)".
```

Comment:
```suggestion
// formatted as "17 days, 4 hours, 5 minutes, 30 seconds".
```

This initial version I had written has total seconds. But reading this PR, I realize it doesn't add much.

Even if the user manually inputted seconds in their UI/CLI, they should have an idea of that duration in human-friendly units, so should be able to match the output here.

Reply:
That was my thought too.